blinkt.go

```diff
@@ -40,6 +47,25 @@ func writeByte(val int) {
 	}
 }
 
+func (bl *Blinkt) InitStopper() {
+
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt)
+	fmt.Println("Press Control + C to stop")
+
+	go func() {
+		for range signalChan {
+			bl.Clear()
+			bl.Show()
+			os.Exit(1)
+		}
+	}()
+}
+
+func (bl *Blinkt) Sleep(ms int) {
```

Comment:
Can we call it "Delay" so it matches the signature of the WiringPi base library? Also, there's no need to attach it to the Blinkt function, it can just float and be exposed.